js/src/cli/apps.ts

```diff
@@ -28,13 +30,20 @@ export default class AppsCommand {
     new AppUpdateCommand(command);
   }
 
-  private async handleAction(options: { browser: boolean }): Promise<void> {
+  private async handleAction(options: {
+    browser: boolean;
+    enabled: boolean;
+  }): Promise<void> {
     getOpenAPIClient();
+    const onlyShowEnabledApps = options.enabled;
     try {
       const { data } = await client.apps.getApps({});
       console.log("Here are the apps you have access to:");
```

Comment:
Consider adding a user-friendly message when filtering is active:
```typescript
console.log("Here are the apps you have access to:");
if (onlyShowEnabledApps) {
  console.log("(Showing enabled apps only)");
}
```
This would improve user experience by making it clear that results are filtered.